### ISSUE TITLE
Verify on Flathub

### ISSUE THREAD

**Issue by starkle** [flathub/com.tutanota.Tutanota#174]
Flathub has a mechanism for verification, which assures users the package is official and not community maintained. Some users wish to exclusively install verified apps, especially for security critical apps such as this one.

Would Tuta be interested in verifying this package? It appears to be a straightforward process, and won't remove the "experimental" disclaimer from the app page.

https://docs.flathub.org/docs/for-app-authors/verification

**Comment by pm4rcin** [comment]
@charlag could you do that?

**Comment by drew585** [comment]
With linux mint (and probably others) hiding unverified flatpacks form the software manager by default, and warning users about installing them, this issue is becoming more important imo.

**Comment by ghostdevv** [comment]
bump 🙏

**Comment by SbMan1** [comment]
Yes, please do this. Thinking of installing Linux Mint for my Aunt since she had a bad Trojan / Virus last week and still getting things ready for her laptop. Since Linux Mint can update Flatpak's automatically in the settings (somewhere), I'd feel more comfortable if this would be verified.   

Thanks for your time.

**Comment by castarco** [comment]
Related, I opened an issue before knowing about this repo: https://github.com/tutao/tutanota/discussions/9731
